pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -438,6 +447,19 @@ func (this *SnapshotManager) deleteSnapshot(peID astrolabe.ProtectedEntityID, ba
 		} else {
 			log.Infof("The upload %v was in terminal stage, proceeding with snapshot deletes", uploadName)
 		}
+		if peID.GetPeType() == "ivd" {
+			log.Infof("Detected delete snapshot of <=1.0.2 backup, using the incoming pe-id %s as is", peID.String())
```

Comment:
Same here.  I think the message should not be so absolute as we are not checking the plugin version.

Reply:
Besides Xing's comment above,

log message: "Detected delete snapshot of backup from plugin prior or equal to v1.0.2 , using the incoming pe-id %s as is"